testsuite/basicA-fork/run-tests.tpl

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+{% include 'common-run.sh.tpl' %}
+
+s2e run -n {{ project_name }}
+
+grep -q "to state 1" $S2E_LAST/debug.txt
```

Comment:
You must make sure that your test case generates 4 states. 2 from the first fork, and 2 from the two other branches.